### ISSUE TITLE
Implement method of changing network keys in EEPROM with downlink

### ISSUE THREAD

**Issue by MedadRufus** [ImperialSpaceSociety/LoRaMac-node#231]
will partially solve worries about expiring credentials expressed in #214 .

An uplink will have to include the following datastructure:

```c
typedef struct
{
    uint8_t FNwkSIntKey[16];
    uint8_t SNwkSIntKey[16];
    uint8_t NwkSEncKey[16];
    uint8_t AppSKey[16];
    uint32_t DevAddr;
    uint32_t frame_count;
    uint32_t Crc32;
} network_keys_t;
```

Also must include which keys it is modifying.

any one of the following:
```c
typedef enum
{
    AS923_KEYS_EU1,
    AS923_KEYS_AU1,
    AU915_KEYS_AU1,
    CN470_KEYS_EU1,
    EU868_KEYS_EU1,
    HELIUM_KEYS,
    KR920_KEYS_EU1,
    IN865_KEYS_EU1,
    US915_KEYS_US1,
    RU864_KEYS_EU1,
    NUMBER_OF_REGISTERED_DEVICES,

} registered_devices_t;

```

**Comment by MedadRufus** [comment]
` uint8_t FNwkSIntKey[16],     uint8_t SNwkSIntKey[16],     uint8_t NwkSEncKey[16]`  are identical, so fewer bytes can be sent to save on bytes used.

**Comment by MedadRufus** [comment]
make it send an ack in the next uplink to confirm that the change has happened

**Comment by MedadRufus** [comment]
#281 tracks the ack issue. But the issue to send keys in uplink was solved in #250.
